### PR TITLE
to-App completed

### PR DESCRIPTION
**Pull request by harikakurma**

### CODE REVIEW

File: src/components/MyHeader.jsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+  handleInputChange = e => {
+    this.setState({
+      ...this.state,
+      input: e.target.value
+    });
+  };
+
+  addTodo = e => {
+    e.preventDefault();
+    /**
```
Comment (theAndrewCline) [inline src/components/MyHeader.jsx]: maybe remove this comment or move it closer to state

File: src/components/MyHeader.jsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+    const filterTask = this.state.tasks.filter(input => input.id !== id);
+    const selectedTask = this.state.tasks.find(input => input.id === id);
+    // console.log(selectedTask);
+    this.setState({
+      tasks: filterTask,
+      input: selectedTask.content,
+      id: id,
+      editItem: true
+    });
+    //console.log(this.state.input);
```
Comment (theAndrewCline) [inline src/components/MyHeader.jsx]: commented console.logs should be removed

**Comment by theAndrewCline** [comment]
Please address minor concerns then I will merge.

File: src/components/MyHeader.jsx (hunk 1 above)
Comment (harikakurma) [reply src/components/MyHeader.jsx]: @theAndrewCline made that change

File: src/components/MyHeader.jsx (hunk 2 above)
Comment (harikakurma) [reply src/components/MyHeader.jsx]: @theAndrewCline ok cleared

**Comment by theAndrewCline** [comment]
@harikakurma I merged your changes in. Good Job!

**Comment by harikakurma** [comment]
@theAndrewCline The more I am learning the more interesting it is thank you for your help.

**Pull request merged**